multisig/proposals/ethereum/vip-019/index.ts

```diff
@@ -0,0 +1,112 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+export const WeETH_ORACLE_EQUIVALENCE = "0xEa687c54321Db5b20CA544f38f08E429a4bfCBc8";
+export const weETH = "0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee";
+export const CHAINLINK_WEETH_FEED = "0x5c9C449BbC9a6075A2c061dF312a35fd1E05fF22";
+export const VTREASURY = "0xfd9b071168bc27dbe16406ec3aba050ce8eb22fa";
+export const vweETH = "0xb4933AF59868986316Ed37fa865C829Eba2df0C7";
```

Comment:
The `protocolShareReserve` contract in the vToken contract should be `0x8c8c8530464f7D95552A11eC31Adbd4dC4AC4d3E`, not the Venus Treasury. I would add a command in the VIP to change it